### ISSUE TITLE
Mixin follow-up

### ISSUE THREAD

**Issue by tobie** [heycam/webidl#450]
- [ ] Reorganize content:
    - [ ] Move namespace up after Mixins and before Members.
    - [ ] Split Interface section into the following subsections:
        - interface
        - inheritance
        - callbacks
        - partial interface
    - [ ] Give subtitles to grammar sections?

**Comment by annevk** [comment]
Are there tooling issues so specifications can start to adopt them? I'd love to adopt them in DOM and Fetch and see what I run into (hopefully nothing).

**Comment by tobie** [comment]
Yeah, I still need to file issues with parsers, Bikeshed and ReSpec, and create a "downstream tracking" issue for this. As there are so many specs touched by this, I was planning to write a small script to file all the issues for me.
